source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.TestCore/InstantHelper.cs

```diff
@@ -48,5 +48,21 @@ public static Instant GetTodayPlusDaysAtMidnightUtc(int noOfDaysToAdd)
         {
             return Instant.FromDateTimeUtc(DateTime.Now.AddDays(noOfDaysToAdd).Date.ToUniversalTime());
         }
+
+        public static Instant GetThisMonthPlusMonthsAtMidnightUtc(int noOfMonthsToAdd)
+        {
+            var startOfMonth = new DateTime(DateTime.Now.Year, DateTime.Now.Month, 1);
+            return Instant.FromDateTimeUtc(DateTime.SpecifyKind(startOfMonth.AddMonths(noOfMonthsToAdd), DateTimeKind.Utc));
```

Comment:
Not sure why this deviates a bit from the other methods?
It returns the first day of a month at time 00:00:00.
I think it should be like this to get the UTC equivalent to local midnight at the first of a month:

```suggestion
            return Instant.FromDateTimeUtc(startOfMonth.AddMonths(noOfMonthsToAdd).Date.ToUniversalTime());
```